api/app/controllers/spree/api/base_controller.rb

```diff
@@ -64,9 +82,9 @@ def load_user
       def authenticate_user
         unless @current_api_user
           if requires_authentication? && api_key.blank? && order_token.blank?
-            render "spree/api/errors/must_specify_api_key", :status => 401 and return
+            render json: { error: I18n.t(:must_specify_api_key, scope: "spree.api") } , status: 401 and return
           elsif order_token.blank? && (requires_authentication? || api_key.present?)
-            render "spree/api/errors/invalid_api_key", :status => 401 and return
+            render json: { error: I18n.t(:invalid_api_key, :key => api_key, :scope => "spree.api") }, status: 401 and return
```

Comment:
Line is too long. [124/100]<br>Use `&&` instead of `and`.<br>Use the new Ruby 1.9 hash syntax.